tensorboard/manager.py

```diff
@@ -315,11 +317,8 @@ def get_all():
     try:
       info = _info_from_string(contents)
     except ValueError:
-      tb_logging.get_logger().warning(
-          "invalid info file: %r",
-          filepath,
-          exc_info=True,
-      )
+      # Ignore unrecognized files.
```

Comment:
We could always log at debug here, if you want.  Up to you.

Reply:
Sure: might as well, and might come in handy.